stereotype/contrib/schematics.py

```diff
@@ -27,10 +40,20 @@ class SchematicsModelField(ModelField):
     :param to_primitive_name: Changes the key used to represent the field in serialized data - output only
     """
 
-    def __init__(self, *, default: Any = Missing, hide_none: bool = False,
-                 primitive_name: Optional[str] = Missing, to_primitive_name: Optional[str] = Missing):
-        super().__init__(default=default, hide_none=hide_none,
-                         primitive_name=primitive_name, to_primitive_name=to_primitive_name)
+    def __init__(
```

Comment:
No need to change it in this PR, but I'd recommend not combining automated reformatting (with tools such as Black) in the same PR as other changes, and never in the same commit. Makes it quite difficult to find the actual changes between all the diffs from those tools. Plus it binds your changes together with the decision to adopt a particular formatting tool.

But again, not really an issue, I'll probably format all the stuff with Black eventually.

Reply:
Ah, now I see you already talked about it in the description, please disregard. :)